plugin/pkg/admission/alwayspullimages/admission.go

```diff
@@ -65,6 +66,27 @@ func (a *AlwaysPullImages) Admit(ctx context.Context, attributes admission.Attri
 	if !ok {
 		return apierrors.NewBadRequest("Resource was marked with kind Pod but was unable to be converted")
 	}
+	oldPod, ok := attributes.GetOldObject().(*api.Pod)
```

Comment:
`getOldObject()` only populates for `Update`. It fails in `Create` operation.

Reply:
fixed. thanks